Yoast/YoastSEO.js
# Added sourcemaps for Browserify

## Summary

This PR introduces separate sourcemaps for both the browserified example and `dist/yoast-seo.js`.

## Relevant technical choices:

Uses `grunt-exorcise` as a wrapper for Exorcise, which is being used based on recommendations in the Browserify documentation.

## Test instructions

This PR can be tested by following these steps:

- Open the browserified example.
- Run `npm install`.
- Add something like a `console.error` to `app.js`.
- Run `grunt build:js`, reload the browserified example and check the console for an error referring to the proper line in `app.js`.
- Remove the `example-browserified.map` file from `examples/browserified/, reload the page and check the console  for a reference to `example-browserified.js`, indicating that the source file couldn't be found.

Fixes https://github.com/Yoast/Wiki/issues/17
